win_build_vs.bat

```diff
@@ -65,34 +63,33 @@ for %%x in (%*) do (
     if [%%~x] EQU [/T]                set skiptests=ON
     if [%%~x] EQU [/R]                set skiptestrun=ON
     if [%%~x] EQU [/P]                set cpack=ON
-    if [%%~x] EQU [/K]                set build_kafka=ON
-    if [%%~x] EQU [/J]                set build_JNI=ON
-    if [%%~x] EQU [/S]                set build_SQL=ON
-    if [%%~x] EQU [/C]                set build_coap=ON
-    if [%%~x] EQU [/A]                set build_AWS=ON
-    if [%%~x] EQU [/SFTP]             set build_SFTP=ON
```

Comment:
what happened to `/SFTP`?

Reply:
Accidentally deleted it, most likely either during the manual conflict resolution with 4e08a94 or somewhere during the automatic rebase process.

Added back in [e859303](https://github.com/apache/nifi-minifi-cpp/pull/1654/commits/e859303f007aff701cccf40b2ef6dab90c1f44c4). I also realized with this change that Windows.md needs to be updated as well, so I did that in [e11810e](https://github.com/apache/nifi-minifi-cpp/pull/1654/commits/e11810edda2c2fd69ac9362c7c344b772b3d640c)